### ISSUE TITLE
[JENKINS-33947] setup wizzard "Dependencies" is not navigable with keyboard

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-06-a2#1468]
<p>start the new wizard and choose customize plugins.</p>

<p>try and expand the dependencies of a plugin without using a mouse/touchscreen</p>

<h3><a name="expectedresults"></a>expected results</h3>

<p>it works</p>

<h3><a name="actualresults"></a>actual results</h3>

<p>epic failure.<br/>
Accessibility issue.</p>

---
<details><summary><i>Originally reported by <a title="James Nord" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=teilo">teilo</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-33947" target="_blank">setup wizzard "Dependencies" is not navigable with keyboard</a></i></summary>
<i><ul>
<li><b>assignee</b>: <a title="Keith Zantow" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=kzantow">kzantow</a>
<li><b>reported by</b>: teilo
<li><b>status</b>: Resolved
<li><b>priority</b>: Major
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2016-05-27T13:29:33+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 3
<li><b>imported</b>: 2025-11-06
</ul></i>
</details>
